doc/lua_api.txt

```diff
@@ -6946,13 +6946,17 @@ object you are working with still exists.
             (default: `true`)
         * `texture`: A regular texture for the sun. Setting to `""`
             will re-enable the mesh sun. (default: "sun.png", if it exists)
+            At sunrise, the texture appears non-rotated on the screen if one
+            looks at it.
```

Comment:
Proposed wording:

The texture appears non-rotated at sunrise and rotated 180 degrees (upside down) at sunset.

Reply:
Sounds good. Done.